enginetest/enginetests.go

```diff
@@ -3648,6 +3684,24 @@ func TestWindowFunctions(t *testing.T, harness Harness) {
 		{5, 1.0},
 	}, nil, nil)
 
+	TestQueryWithContext(t, ctx, e, harness, `SELECT a, rank() over (partition by c order by b) FROM t1 order by a`, []sql.Row{
```

Comment:
Windows have a lot of edge cases, need to test different combinations of partition, order, framing for each.